### DISCUSSION TITLE
Subestructura óptima del problema - Parte 1

### DISCUSSION THREAD

**Discussion by vnloyola** [IIC2133-PUC/2024-2#178] [Tarea 3] [answered]
Hola! Tengo una duda acerca de lo que se pide responder en el informe sobre la subestructura óptima. 

En la pregunta del informe *¿Por qué el problema sigue la subestructura óptima?* Se pide demostrar de manera formal que el problema la siga o se debe explicar de manera simplificada que la sigue?

Muchas gracias!

**Comment by tomastrivino** [comment, marked as answer]
No hace falta usar métodos formales como la inducción

Saludos!
